process/process_areas/quality_management/quality_roles.rst

```diff
@@ -0,0 +1,59 @@
+..
+   # *******************************************************************************
+   # Copyright (c) 2025 Contributors to the Eclipse Foundation
+   #
+   # See the NOTICE file(s) distributed with this work for additional
+   # information regarding copyright ownership.
+   #
+   # This program and the accompanying materials are made available under the
+   # terms of the Apache License Version 2.0 which is available at
+   # https://www.apache.org/licenses/LICENSE-2.0
+   #
+   # SPDX-License-Identifier: Apache-2.0
+   # *******************************************************************************
+
+Roles
+#####
+
+.. role:: Quality Manager
+   :id: rl__quality_manager
+   :status: valid
+
+   The quality managers shall be responsible for the planning and coordination of the quality activities, i.e. the quality management. He/She shall lead and monitor the quality relevant activities of the project.
+
+
+   Required skills
+
+   * Degree: Master degree in software quality management, or similar technical/economical degree, or comparable work experience
```

Comment:
I know about quality management degrees, and I know about software quality management courses as part of a MSc or BSc study, but are there really full time study programs on software quality management? And I would judge that an "computer science" degree is not a similar degree. Maybe this statement can be a bit relaxed as the things below already set the scene.